app/javascript/crayons/Tabs/Tabs.jsx

```diff
@@ -0,0 +1,35 @@
+import { h, cloneElement } from 'preact';
+import classNames from 'classnames/bind';
+
+export const Tabs = (props) => {
+  const {
+    children,
+    stacked,
+    fitted,
+    className,
+    elements = 'buttons',
+    ...otherProps
+  } = props;
+
+  const Wrapper = elements === 'buttons' ? 'div' : 'nav';
+
+  const classes = classNames('c-tabs', {
+    'c-tabs--stacked': stacked,
+    'c-tabs--fitted': fitted,
+    [className]: className,
+  });
+
+  return (
+    <Wrapper className={classes} {...otherProps}>
```

Comment:
We should also require a 'title' or 'name' prop, and add it to the `<nav>` element using an `aria-label`. Labelling the `nav`s helps screen reader users get to the right one quickly and easily 🙂